src/sortMethods.js

```diff
@@ -0,0 +1,107 @@
+
+const sortMethods = {
+
+        SELECT : {
+            name: null,
+            userMessage: "Select one of the below", 
+            comparator: null,
+        },
+
+        TITLE_ASCENDING : {
+            name: "TITLE_ASCENDING",
+            userMessage: "Title A-Z", 
+            category: "Title",
+            comparator: function(a,b) {
+                const movieA = a['Title']; // const movieA = a[this.category] does not work when imported? 
+                const movieB = b['Title'];           
+                let comparison = 0;
+
+                if (movieA > movieB) {
+                    comparison = 1
+                } else if (movieA < movieB) {
+                    comparison = -1
+                };
+    
+                return comparison;
+            },
+            testPrint() {
+                console.log(this.category); 
+            }
+        },
+
+        TITLE_DESCENDING : {
+            name: "TITLE_DESCENDING",
+            userMessage: "Title Z-A", 
+            category: "Title",
+            comparator: function(a,b) {
```

Comment:
it would be possible to extract a function from all the comparators, that takes the property name. You could then invert it for descending functions in each comparator as required. e.g.:

```
function (a, b, propertyName) {
    const movieA = a[propertyName];
    const movieB = b[propertyName];
    // compare as needed here
    return comparison;
}
```

Reply:
Sorry, I'm not sure I understand! I can happily set the sortMethod parameter to reference the propertyName (as shown) can't quite work out how to reference that parameter upon the array.sort() method call?